pubtools/_quay/command_executor.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import shlex
+import subprocess
+
+import paramiko
+
+LOG = logging.getLogger("pubtools-quay")
+LOG.setLevel(logging.INFO)
+
+
+class Executor(object):
+    """
+    Base executor class.
+
+    Implementation of command execution should be done in
+    descendant classes. Common pre- and post-processing should be
+    implemented in this class.
+    """
+
+    def __init__(self):
```

Comment:
:eyes: 

why do you need abstract `__init__` here? It's already part of `object`, python stdlib

Reply:
You're right, I removed it.